src/devices/Mcp23xxx/samples/Mcp23xxx.Samples.csproj

```diff
@@ -6,13 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!--
-        Temporarily making a project reference until the daily has IGpioController
-        <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> 
-    -->
```

Comment:
you can remove the above PackageReference to System.Device.Gpio package as well, as the project reference should bring in the gpio library as a transitive reference.